### PR TITLE
feat(lapis): adapt to new generalized lineage system

### LINKED ISSUE #961: Adapt to generalized lineage queries
https://github.com/GenSpectrum/LAPIS-SILO/issues/458 introduces a couple of breaking changes in SILO. 

Adapt LAPIS accordingly.

### PR DESCRIPTION
**Pull request by fengelniederhammer**
resolves #961

See https://github.com/GenSpectrum/LAPIS-SILO/issues/458. Requires the corresponding PR (https://github.com/GenSpectrum/LAPIS-SILO/pull/590) to get green tests.

Depends on 

## PR Checklist
~~- [ ] All necessary documentation has been adapted.~~
- [x] The implemented feature is covered by an appropriate test.

### CODE REVIEW

**Review by JonasKellerer — APPROVED** [review]
LGTM

**Comment by fengelniederhammer** [comment]
> Do we need to abapt the docs for the config as well?

Yes, I created https://github.com/GenSpectrum/LAPIS/issues/978. I supposed that it's a bigger change, and it should actually be done in the SILO docs, so I didn't want to do it here.

**Comment by JonasKellerer** [comment]
I just looked at it: do we want to adapt the schema for lineage queries, so that it shows more information than for a generic string?

**Comment by fengelniederhammer** [comment]
> I just looked at it: do we want to adapt the schema for lineage queries, so that it shows more information than for a generic string?

I added this:
grafik

Is that what you meant?

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ Higher versions will also work if they are not specified in the table.
 
 | LAPIS  | SILO   |
 |--------|--------|
+| 0.3.5  | 0.3.0  |
```
Comment (fengelniederhammer) [inline README.md:46]: Make sure this is still up to date when merging

**Pull request merged**